app/views/jobseekers/job_applications/build/ask_for_support.html.slim

```diff
@@ -31,4 +31,4 @@
         = f.govuk_submit t("buttons.save_and_come_back"), secondary: true
 
   .govuk-grid-column-one-third
-    = render(Shared::ProcessStepsComponent.new(process: vacancy, service: process_steps, title: t("jobseekers.job_applications.build.process_title")))
+    = render "job_application_steps"
```

Comment:
I dont understand this, is it a WIP thing?

Reply:
I put the rendering of the component in a partial to avoid rendering it in each step.